org.gnome.Eolie.json

```diff
@@ -0,0 +1,314 @@
+{
+  "app-id": "org.gnome.Eolie",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "command": "eolie",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=host",
```

Comment:
Does a web browser really need host access? If it uses GtkFileChooserNative that should cover most usage.